frontend/app/src/main/java/testdata/DummyAccountRepository.kt

```diff
@@ -0,0 +1,35 @@
+package testdata
+
+import dk.scheduling.schedulingfrontend.repositories.account.IAccountRepository
+import java.util.UUID
+
+class DummyAccountRepository : IAccountRepository {
+    override suspend fun getAuthToken(): UUID {
+        return UUID.randomUUID()
```

Comment:
Maybe not return a new random UUID every time we get the auth token. Seems pretty unexpected.